os.optaweb/module.yaml

```diff
@@ -0,0 +1,6 @@
+schema_version: 1
+name: os.optaweb
```

Comment:
is this intended? This is optaweb-employee-rostering, but we'll also have optaweb-vehicle-routing (a completely separate app) in the future. We don't want names to collide.

Reply:
Because this file holds employee-rostering specific values, I recommend renaming the directory "os.optaweb" to "os.optaweb.employeerostering"